pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedPrivateMethodRule.java

```diff
@@ -51,29 +52,38 @@ protected Collection<String> defaultSuppressionAnnotations() {
 
     @Override
     public Object visit(ASTCompilationUnit file, Object param) {
-        // this does a couple of traversals:
-        // - one to find annotations that potentially reference a method
-        // - one to collect candidates, that is, potentially unused methods
-        // - one to walk through all possible usages of methods, and delete used methods from the set
```

Comment:
kind of dry `one to`

<img width="623" alt="image" src="https://github.com/user-attachments/assets/786cd876-ba17-4e81-8f20-fbdf03046ff8" />

computer can do it for us: 
<img width="692" alt="image" src="https://github.com/user-attachments/assets/aac53208-9581-4050-a6ef-ecf41423f9be" />

relates to: https://github.com/pmd/pmd/pull/5620#discussion_r2020124973